### DISCUSSION TITLE
Is it possible to manually set an Endpoint/device offline ?

### DISCUSSION THREAD

**Discussion by MatthieuCa** [matter-js/matter.js#1047] [Q&A] [answered]
Hello everyone !

I need your help.
I want to know if it is possible to set devices on Home Kit, Google Home, etc. to offline manually ?

If yes, what parameter or function should I use for ?

Thank you,
Matthieu

**Comment by Apollon77** [comment, marked as answer]
Offline aka "unreachable but still leave it included"?

then this should be (IF reachable was used when initializing the server!)

```
serverNode.set({basicInformation: { reachable: false}});
```

or for a bridged endpoint use

```
endpoint.set({bridgedDeviceBasicInformation: { reachable: false }});
```

I honestly did not tested this, so please check if it works. if not I will look deeper.

**Reply by MatthieuCa** [reply]
It works only if you set the reachable during the creation of the endpoint, this is not working if you want to set this to offline at another time.

For me it's not a problem because that's what I wanted, but if someone wants to set the reachable parameters after the initialisation it will be more difficult I think

**Reply by MatthieuCa** [reply]
My mistake the set work also after initialisation sorry

**Reply by Apollon77** [reply]
... thank you for verifying ...I was just about to ask why it is not working ;-)
